pkg/gh2gcs/gh2gcs.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gh2gcs
+
+import (
+	"path/filepath"
+
+	"k8s.io/release/pkg/gcp/gcs"
+	"k8s.io/release/pkg/github"
+)
+
+type Config struct {
```

Comment:
What is the main intention to have another wrapper struct here? Can we just use `ReleaseConfig` (or rename `ReleaseConfig` to `Config`)?

Reply:
This is for future functionality. Added the following doc:
```go
// Config contains a slice of `ReleaseConfig` to be used when unmarshalling a
// yaml config containing multiple repository configs.
```

Reply:
Thanks!